moor_generator/lib/src/writer/data_class_writer.dart

```diff
@@ -94,6 +97,17 @@ class DataClassWriter {
     buffer.write(');}\n');
   }
 
+  void _writeToJson(StringBuffer buffer) {
+    buffer.write('Map<String, Object> toJson() {\n return {');
```

Comment:
Might be a personal thing, but I'd prefer this to return `Map<String, dynamic>`, it feels more appropriate as that's also what would be returned by `json.decode`.
Unless you have a better argument in favor of `Object`, of course :)